cardano-testnet/src/Testnet/Util/Runtime.hs

```diff
@@ -108,6 +123,22 @@ bftSprockets = fmap nodeSprocket . bftNodes
 poolSprockets :: TestnetRuntime -> [Sprocket]
 poolSprockets = fmap (nodeSprocket . poolRuntime) . poolNodes
 
+shelleyGenesis :: (H.MonadTest m, MonadIO m, HasCallStack) => TestnetRuntime -> m (ShelleyGenesis StandardCrypto)
+shelleyGenesis TestnetRuntime{shelleyGenesisFile} = withFrozenCallStack $
+  H.evalEither =<< H.evalIO (A.eitherDecodeFileStrict' shelleyGenesisFile)
+
+getStartTime :: (H.MonadTest m, MonadIO m, HasCallStack) => FilePath -> TestnetRuntime -> m UTCTime
+getStartTime tempRootPath TestnetRuntime{configurationFile} = withFrozenCallStack $ H.evalEither <=< H.evalIO . runExceptT $ do
+  testnetConfig <- ExceptT $ A.eitherDecodeFileStrict' configurationFile
```

Comment:
So what we should do here is decode a `NodeConfiguration`. We can then retrieve the `NodeProtocolConfiguration` which will contain `NodeByronProtocolConfiguration` which has the Byron genesis file path.

Reply:
There's no `FromJSON NodeByronProtocolConfiguration` , and it seems unnecessary to maintain one for just this case.